packages/icm20948-1.0.0.yaml

```diff
@@ -0,0 +1,5 @@
+name: icm20948
+description: Driver for ICM-20948 9-axis devices.
```

Comment:
minor nit, but they should/must agree: The description here finishes with a ".", but doesn't in the README.